### ISSUE TITLE
Advanced HTML and CSS: Update codeblock delimiters

### ISSUE THREAD

**Issue by aconstas** [TheOdinProject/curriculum#27299]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

Codeblocks should be updated to align with new changes in our [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

In the sections named in this issue, codeblocks will be opened by a set of 3 tildes (the ~ character) and then closed by another set of 3 tildes. To align with the style guide, these tilde characters need to change to backticks (the ` character). Be sure to refer to [the codeblocks section](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#codeblocks) of the style guide for an example of what this should look like.
## Guidance:
- Comment on this issue to get a section assigned to you. You can also ask for a specific one
- Please wait until you are assigned by a maintainer before starting to work on this issue to avoid multiple people working on the same sections
- Make sure to read the [contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md) before starting your first contribution
- Please add the following to your PR in the Issues section: Related to #xxxxx, since this issue is divided into sections, individual PRs should NOT INCLUDE the closes keyword.
- **Please ensure that you update the delimiters in every listed lesson of the section you are assigned to.**

### [Accessibility (4 lessons)](https://www.theodinproject.com/paths/full-stack-javascript/courses/advanced-html-and-css#accessibility)
- [x] @roopxx ***Keyboard Navigation, Meaningful Text, Semantic HTML, WAI Aria***

### [Animation (3 lessons)](https://www.theodinproject.com/paths/full-stack-javascript/courses/advanced-html-and-css#animation)
- [x] @Churtified ***Keyframes, Transforms, Transitions***

### [Responsive Design (1 lesson)](https://www.theodinproject.com/paths/full-stack-javascript/courses/advanced-html-and-css#responsive-design)
- [x] @roopxx ***Media Queries***

@JoshDevHub

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/paths/full-stack-javascript/courses/advanced-html-and-css

**Comment by roopxx** [comment]
I'm happy to take one section.

**Comment by JoshDevHub** [comment]
Awesome @roopxx 

I've assigned you to the Accessibility section.

**Comment by Churtified** [comment]
I can take one :)

**Comment by JoshDevHub** [comment]
Thank you for volunteering @Churtified 

I've assigned you the animation section :thumbsup:

**Comment by roopxx** [comment]
@JoshDevHub, I can take the last section too :-)

**Comment by JoshDevHub** [comment]
Thanks for volunteering for another @roopxx 

I've assigned you the responsive design section.

**Comment by JoshDevHub** [comment]
Alright all issues have been solved. Thank you for contributing everyone!
